src/components/05_pages/NodeEditForm/index.js

```diff
@@ -32,21 +38,18 @@ NodeEditForm.defaultProps = {
 };
 
 NodeEditForm.propTypes = {
-  bundle: PropTypes.string.isRequired,
-  id: PropTypes.string.isRequired,
+  nid: PropTypes.string.isRequired,
   requestSingleContent: PropTypes.func.isRequired,
   entity: PropTypes.shape({
-    data: PropTypes.shape({
-      attributes: PropTypes.shape({
-        title: PropTypes.string.isRequired,
-      }).isRequired,
+    attributes: PropTypes.shape({
```

Comment:
It feels like a follow up could expand the prop type a bit more to include relationships as well?

Reply:
Yes. I want to create a followup to address PropTypes across all our components.